### PR TITLE
docs: update parameters.md coding guidelines

### PR DESCRIPTION
**Pull request by kaspermeck-arm**
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/contributing/coding-guidelines/ros-nodes/parameters.md [hunk 1]
```diff
@@ -99,16 +99,17 @@ Note: `/**` is used instead of the explicit node namespace, this allows the para
           "type": "INSERT_TYPE",
           "description": "INSERT_DESCRIPTION",
           "default": "INSERT_DEFAULT",
```
Comment (ambroise-arm) [inline docs/contributing/coding-guidelines/ros-nodes/parameters.md:101]: Do we keep the quotes for the default value? For a string or a boolean for example it takes quotes, but not for integers.

Comment (kaspermeck-arm) [reply docs/contributing/coding-guidelines/ros-nodes/parameters.md:101]: What I noticed is that it populates the parameter file correctly if the "default" value is within quotes. E.g., it'll keep `10.0` and not populate `10`. The validation still works in VSCode.

**Review by esteve — APPROVED** [review]
@kaspermeck-arm thanks!

**Review by esteve — CHANGES_REQUESTED** [review]
@kaspermeck-arm changes look ok, but this PR is failing the DCO check. Could you sign each of your commits with `git commit -s` ? Thanks.

File: docs/contributing/coding-guidelines/ros-nodes/parameters.md (hunk 1 above)
Comment (ambroise-arm) [reply docs/contributing/coding-guidelines/ros-nodes/parameters.md:101]: Ah, I see. `default` is an annotation, which "_aren’t strictly used for validation, but are used to describe parts of a
schema_". And "_default is typically used to express that if a value is missing, then the
value is semantically the same as if the value was present with the default value_". So having quotes makes perfect sense.

**Comment by kaspermeck-arm** [comment]
> @kaspermeck-arm changes look ok, but this PR is failing the DCO check. Could you sign each of your commits with `git commit -s` ? Thanks.

@esteve - done!

**Comment by kaspermeck-arm** [comment]
@esteve - I addressed the [DCO issue](https://github.com/autowarefoundation/autoware-documentation/pull/434/checks?check_run_id=15994451686), can you take a look now?

**Comment by esteve** [comment]
@kaspermeck-arm thanks. I've updated the title of the PR so that it follows our conventions (merging was blocked because of that). I believe you'd have to update the comment in the first commit to match it, but not entirely sure. I've rebased the PR, which triggered another CI run, we'll see if updating the title is enough.

**Review by esteve — APPROVED** [review]

**Comment by esteve** [comment]
@kaspermeck-arm it seems that the DCO check doesn't work well if the changes had been committed via the GitHub website and you have your email address masked. Anyway, I've marked the check. Thanks!

**Comment by kaspermeck-arm** [comment]
@esteve - I see, I'll keep that in mind for next time! Thanks for reviewing and merging :)

**Pull request merged**